h/auth/policy.py

```diff
@@ -352,14 +353,19 @@ def _is_client_request(request):
     """
     Is client_auth authentication valid for the given request?
 
-    For initial rollout, authentication should be performed by
+    Uuthentication should be performed by
```

Comment:
yeah, yeah, I see the typo. I'll fix it in my next round of fixups for this branch.